cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,631 @@
+from widgetastic_manageiq import MigrationDashboardStatusCard
+from widgetastic_manageiq import MigrationDropdown
+from widgetastic_manageiq import MigrationPlanRequestDetailsList
+from widgetastic_manageiq import MigrationPlansList
+from widgetastic_manageiq import MigrationProgressBar
+from widgetastic_manageiq import RadioGroup
+from widgetastic_manageiq import Table
+from widgetastic_manageiq import V2VPaginatorPane
+
+
+class MigrationView(BaseLoggedInPage):
+    title = Text("#explorer_title_text")
+
+    @property
+    def in_explorer(self):
+        nav_menu = VersionPicker(
+            {
+                Version.lowest(): ["Compute", "Migration"],
+                "5.10": ["Compute", "Migration", "Migration Plans"],
+            }
+        )
+
+        return self.logged_in_as_current_user and self.navigation.currently_selected == nav_menu
+
+    @property
+    def is_displayed(self):
+        return self.in_explorer
+
+
+class DashboardCards(View):
+    not_started_plans = MigrationDashboardStatusCard("Not Started")
+    in_progress_plans = MigrationDashboardStatusCard("In Progress")
+    completed_plans = MigrationDashboardStatusCard("Complete")
+    archived_plans = VersionPicker(
+        {
+            Version.lowest(): MigrationDashboardStatusCard("Infrastructure Mappings"),
+            "5.10": MigrationDashboardStatusCard("Archived"),
+        }
+    )
+
+    def read(self):
+        return [c.read().get("name") for c in self.sub_widgets if c.is_active]
+
+
+class MigrationPlanView(MigrationView):
+    dashboard_cards = View.nested(DashboardCards)
+    plans_list = ConditionalSwitchableView(reference="dashboard_cards")
+    paginator_view = View.include(V2VPaginatorPane, use_parent=True)
+    title = Text('.//div[contains(@class, "pull-left")]//h3')
+    create_migration_plan = Text(locator='(//a|//button)[text()="Create Migration Plan"]')
+    configure_providers = Text(locator='//a[text()="Configure Providers"]')
+    sort_type_dropdown = SelectorDropdown("id", "sortTypeMenu")
+    sort_direction = Text(locator=".//span[contains(@class,'sort-direction')]")
+    # TODO - fix this once dev fixes the locator
+    progress_card = MigrationProgressBar(locator=".//div[3]/div/div[3]/div[3]/div/div")
+    search_box = TextInput(locator=".//div[contains(@class,'input-group')]/input")
+    clear_filters = Text(".//a[text()='Clear All Filters']")
+    # 5.9 field
+    migr_dropdown = MigrationDropdown(text="Not Started Plans")
+
+    @property
+    def in_migration_plan(self):
+        return (
+            len(self.browser.elements('.//div[contains(@class,"card-pf")]')) > 0
+            and len(self.browser.elements(PFIcon.icons.WARNING)) < 1
+        )
+
+    @property
+    def is_displayed(self):
+        return self.in_migration_plan
+
+
+class NotStartedPlansView(MigrationPlanView):
+    TITLE = "Not Started"
+    plans_not_started_list = MigrationPlansList("plans-not-started-list")
+
+    @property
+    def is_displayed(self):
```

Comment:
I think this `is_displayed` should follow the pattern with what we normally do when we have a change between versions. Something like:

```
 @property
    def is_displayed(self):
        expected_title = 'Not Started Plans'
        on_sub_page = self.in_migration_plan and self.plans_not_started_list.is_displayed

        if self.context['obj'].appliance.version >= '5.10':
            return on_sub_page and self.title.text == expected_title
        else:
            return on_sub_page
```